extension/encodingextension/metadata.yaml

```diff
@@ -1,8 +1,9 @@
----
 type: encoding
 
 status:
   class: extension
   stability:
     development: [extension]
-  distributions: []
\ No newline at end of file
+  distributions: []
```

Comment:
Is this correct? Isn't this part of any distributions?

Reply:
yes, as this is still in development.